package.json

```diff
@@ -77,18 +77,13 @@
     "prettier": "prettier --list-different '{.storybook,packages}/**/*.{ts,tsx}'",
     "prettier-clean": "prettier --write '{.storybook,packages}/**/*.{ts,tsx}'",
     "lint": "yarn pre-test && eslint --ext .tsx,.ts ./packages",
-    "type-check": "lerna run tsc",
     "tsc": "echo 'Did you mean to run yarn type-check?'",
     "chromatic": "chromatic",
     "packages:check": "manypkg check",
     "packages:fix": "manypkg fix",
     "changeset": "changeset",
     "create:package": "plop create-package",
-    "token-usage": "ts-node tools/build/token-usage-detector.ts",
-    "turbo-build": "turbo run build",
-    "turbo-test": "turbo run test",
```

Comment:
I wanted to add turborepo to other commands such as `test` `lint` `prettier` and `type-check`, but couldn't get it to work after a couple hours, so I decided to work with the main pain point for now (builds) and approach this again later.